metrics/firebase/functions/test/main_test.dart

```diff
@@ -16,65 +16,74 @@ import 'test_utils/test_data/build_test_data_generator.dart';
 // ignore_for_file: avoid_redundant_argument_values
 
 void main() {
-  group("onBuildAddedHandler", () {
-    const tasksCollectionName = 'tasks';
-    const projectId = 'projectId';
-    const durationInMilliseconds = 123;
-    const buildStatus = BuildStatus.successful;
-
-    final startedAt = DateTime.now();
-    final startedAtDayUtc = startedAt.toUtc().date;
-    final testDataGenerator = BuildTestDataGenerator(
-      projectId: projectId,
-    );
+  const tasksCollectionName = 'tasks';
+  const projectId = 'projectId';
+  const durationInMilliseconds = 123;
+  const buildStatus = BuildStatus.successful;
+
+  final startedAt = DateTime.now();
+  final startedAtDayUtc = startedAt.toUtc().date;
+  final testDataGenerator = BuildTestDataGenerator(
+    projectId: projectId,
+  );
+  final buildJson = testDataGenerator.generateBuildJson();
```

Comment:
I think it would be great to use this variable in the `creates a task document with data equals to the build data if setting the build day's document data fails` test to avoid duplication.